fuzz/fuzz_targets/streams.rs

```diff
@@ -0,0 +1,71 @@
+#![no_main]
+
+use arbitrary::{Arbitrary, Unstructured};
+use libfuzzer_sys::fuzz_target;
+
+extern crate proto;
+use proto::fuzzing::{ResetStream, Streams};
+use proto::{Dir, Side, StreamId, VarInt};
+
+#[derive(Arbitrary, Debug)]
+struct StreamParams {
+    side: Side,
+    max_remote_uni: u16,
+    max_remote_bi: u16,
+    send_window: u16,
+    receive_window: u16,
+    stream_receive_window: u16,
+    dir: Dir,
+}
+
+#[derive(Arbitrary, Debug)]
+enum Operation<A>
+where
+    A: Arbitrary,
+{
+    Open(Vec<A>),
```

Comment:
Is there a benefit to this being generic rather than hardcoding `u8`?

Reply:
no you're right that's not needed - removing.

Reply:
No that is no longer useful; removed